templates/cookiepolicy/services.php

```diff
@@ -2,7 +2,8 @@
 	<summary class="cmplz-service-header"><div>
 		<h3>{service}</h3>
 		<p>{allPurposes}</p>
-		<input type="checkbox" class="cmplz-accept-service {serviceCheckboxClass}" data-service="{service_slug}" data-category="{topCategory}"></div>
```

Comment:
This checkbox cannot get a service visible before it, as it is shown right after the purposes. Perhaps the label can encompass both h3 and p, but I expect that this is not allowed (using html elements in a label). The current change is not possible however. See for example the use of this checkbox in https://complianz.io/legal/cookie-policy/?cmplz_region_redirect=true&cmplz-region=eu.

Reply:
Indeed it could not be possible to include html elements in the `label`. 
To make it more interesting, the variable `serviceCheckboxClass` receives the value `cmplz-hidden`!

However, unless `achecker` shows "Congratulations" we will not get the fund. 
[`WPML`](https://wpml.org/errata/missing-alt-tag-for-flag-image-in-a-language-switcher/) misses an `alt` by design purely for accessibility reasons, we had to add it for the `checker` to stop complaining, it is insane!

Reply:
Indeed, the serviceCheckboxClass is hidden by default. This is because it is only visible when consent per service is activated in the wizard. The scan not detecting this shows the problem with such scans: the scans are viewed as the holy grail but often show wrong conclusions. 

We will look at your other suggestions, but won't be able to include this one in the current form. You can always override the template for this snippet if necessary. Let me know if you need any help with this.

Reply:
OK, thank you!